tests/tests_integration/test_session/test_graph_flow/test_classic_to_dms.yml

```diff
@@ -2821,6 +2830,15 @@ rules:
     value_type: json
     view: ClassicSequence
     view_property: rows
+  - container: ClassicSourceSystem
+    container_property: classicSourceSystemGUID
+    immutable: false
+    is_list: false
+    neatId: http://purl.org/cognite/neat/neatId_baba86c4_fed1_4faf_8e48_b5f1e8179a50
+    nullable: true
+    value_type: text
+    view: ClassicSourceSystem
+    view_property: classicSourceSystemGUID
```

Comment:
This is the resulting new property. This ensures that only instances written through this view will show up in it again.